trace_decoder/src/compact/compact_prestate_processing.rs

```diff
@@ -1453,6 +1466,24 @@ mod tests {
         hex::decode(b_str).unwrap()
     }
 
+    // TODO: Move behind a feature flag just used for debugging (but probably not
+    // `debug`)...
+    fn parse_just_to_instructions(bytes: Vec<u8>) -> CompactParsingResult<Vec<Instruction>> {
```

Comment:
Incredibly picky but what does "just" add in the fn name?

Reply:
So I'll post a more detailed writeup below, but I created a handful of methods/functions where the intent is just for someone debugging output the user generated with their own tool and are passing to the decoder. Like here, they may want to verify that the instructions we're decoded properly (nit: we probably should remove `parse` from these names), but it never makes sense to call this in production, so I thought we should move these behind a feature flag to reduce the binary size a bit.